k8s/europe-west4/profile-cleanup.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: profile-cleanup
+spec:
+  schedule: "0 0 * * *"
```

Comment:
Any chance this could cause problem if run concurrently with a pods test? I'm assuming its fairly unlikely?

Reply:
I think it's pretty unlikely, but I'll keep an eye out for issues.